.github/workflows/pr-update.yaml

```diff
@@ -0,0 +1,21 @@
+name: PR update
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
Can we also enable this for `release-*`?

Reply:
Will add.